### DISCUSSION TITLE
I need whisper to make smaller subtitle excerpts.

### DISCUSSION THREAD

**Discussion by Pokoyo22421** [openai/whisper#2233] [Q&A] [answered]
My whisper is working perfectly, however I would like to use it to subtitle some videos, when I enter the "srt" subtitle files for example, each excerpt becomes very long and with a lot of text, this way the subtitled video becomes polluted, I would like the whisper put a limit of 3 seconds for each excerpt it transcribes or even a text limit per excerpt, for example 8 words per excerpt, is this possible?

**Comment by glangford** [comment, marked as answer]
See `whisper --help` and these options

```
--max_line_width MAX_LINE_WIDTH
                        (requires --word_timestamps True) the maximum number of characters in a line before breaking the line (default: None)
  --max_line_count MAX_LINE_COUNT
                        (requires --word_timestamps True) the maximum number of lines in a segment (default: None)
  --max_words_per_line MAX_WORDS_PER_LINE
                        (requires --word_timestamps True, no effect with --max_line_width) the maximum number of words in a segment (default: None)
```

**Reply by Pokoyo22421** [reply]
Thanks for showing me, this was exactly what I was looking for, however when placing the command:
whisper "video.mp4" --language en --word_timestamps True --max_words_per_line 8"

I receive the following error:
C:\Users\usuario\Desktop\Videos>whisper "video.mp4" --language en --word_timestamps True --max_words_per_line 8
C:\Users\usuario\AppData\Local\Programs\Python\Python312\Lib\site-packages\whisper\timing.py:42: UserWarning: Failed to launch Triton kernels, likely due to missing CUDA toolkit; falling back to a slower median kernel implementation...
warnings.warn(
C:\Users\usuario\AppData\Local\Programs\Python\Python312\Lib\site-packages\whisper\timing.py:146: UserWarning: Failed to launch Triton kernels, likely due to missing CUDA toolkit; falling back to a slower DTW implementation...
warnings.warn(

I tried to use gpt chat to help me but it said to check if cuda is installed and in the path, and when I enter the command I check that it is working:
C:\Users\usuario\Desktop\Videos>nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2024 NVIDIA Corporation
Built on Wed_Apr_17_19:36:51_Pacific_Daylight_Time_2024
Cuda compilation tools, release 12.5, V12.5.40
Build cuda_12.5.r12.5/compiler.34177558_0



Edit: Even with the error message the output.srt file follows the 8 words per line rule, thanks!

**Comment by glangford** [comment]
> C:\Users\usuario\AppData\Local\Programs\Python\Python312\Lib\site-packages\whisper\timing.py:146: UserWarning: Failed to launch Triton kernels, likely due to missing CUDA toolkit; falling back to a slower DTW implementation

This is just a warning, not an error, likely because triton is not supported on Windows. See

* https://github.com/openai/whisper/discussions/1283#discussioncomment-5813440
